actool/manifest.go

```diff
@@ -422,7 +423,7 @@ func runPatchManifest(args []string) (exit int) {
 		}
 	}
 
-	err = extractManifest(tr, tw, false, newManifest)
+	err = extractManifest(tr.Reader, tw, false, newManifest)
```

Comment:
Unfortunately when a functions wants a *tar.Reader, passing the embedding struct doesn't work. I haven't found a better way to avoid this...